I believe this is an issue upstream. I am getting a bunch of errors like

```
==> Starting build()...
npm warn deprecated prebuild-install@7.1.3: No longer maintained. Please contact the author of the relevant native addon; alternatives are available.
npm warn deprecated node-domexception@1.0.0: Use your platform's native DOMException instead

added 355 packages in 5s
npm notice run @earendil-works/pi-tui@0.81.0 build
npm notice run tsgo -p tsconfig.build.json
packages/ai/src/providers/amazon-bedrock.models.ts:4:20 - error TS2307: Cannot find module './data/amazon-bedrock.json' or its corresponding type declarations.

4 import values from "./data/amazon-bedrock.json" with { type: "json" };
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

packages/ai/src/providers/ant-ling.models.ts:4:20 - error TS2307: Cannot find module './data/ant-ling.json' or its corresponding type declarations.

4 import values from "./data/ant-ling.json" with { type: "json" };

... SNIP ...

Found 37 errors in 37 files.

Errors  Files

... SNIP ...
```

--- a/pi/.SRCINFO
+++ b/pi/.SRCINFO
@@ -12,6 +12,8 @@ pkgbase = pi
 	optdepends = fd: system-provided backend for the find tool
 	optdepends = ripgrep: system-provided backend for the grep tool
 	source = pi-0.81.0.tar.gz::https://github.com/earendil-works/pi/archive/refs/tags/v0.81.0.tar.gz
+	source = pi-ai-0.81.0.tgz::https://registry.npmjs.org/@earendil-works/pi-ai/-/pi-ai-0.81.0.tgz
 	sha256sums = f62a706210b98ab368db5d6ef8442aa6bb9ad5b3b73fe71cad02b29cb64e8321
+	sha256sums = 62e0abc1cffac431d091cbc8e88eeefcb4c428bc78d2702ec52bcf5defe1ff5a
 
 pkgname = pi

--- a/pi/PKGBUILD
+++ b/pi/PKGBUILD
@@ -14,20 +14,24 @@ optdepends=(
   'fd: system-provided backend for the find tool'
   'ripgrep: system-provided backend for the grep tool'
 )
-source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/v${pkgver}.tar.gz")
-sha256sums=('f62a706210b98ab368db5d6ef8442aa6bb9ad5b3b73fe71cad02b29cb64e8321')
+
+source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/v${pkgver}.tar.gz"
+        "pi-ai-${pkgver}.tgz::https://registry.npmjs.org/@earendil-works/pi-ai/-/pi-ai-${pkgver}.tgz")
+sha256sums=('f62a706210b98ab368db5d6ef8442aa6bb9ad5b3b73fe71cad02b29cb64e8321'
+            '62e0abc1cffac431d091cbc8e88eeefcb4c428bc78d2702ec52bcf5defe1ff5a')
+
+prepare() {
+  rm -rf "${srcdir}/${pkgname}-${pkgver}/packages/ai/src/providers/data"
+  cp -a "${srcdir}/package/dist/providers/data" \
+    "${srcdir}/${pkgname}-${pkgver}/packages/ai/src/providers/"
+}
 
 build() {
   cd "${pkgname}-${pkgver}"
 
   npm ci --cache "${srcdir}/npm-cache" --ignore-scripts --no-audit --no-fund
 
-  # Running run build specifically for each package is necessary to prevent ai package issue
-  npm --prefix packages/tui run build
-  # This is necessary to prevent ai module fetching models on network in building
-  ./node_modules/.bin/tsgo -p packages/ai/tsconfig.build.json
-  npm --prefix packages/agent run build
-  npm --prefix packages/coding-agent run build
+  npm run build:offline
 
   # Remove packages which are only necessary in development / building
   npm prune --omit=dev --cache "${srcdir}/npm-cache"

